gridstatus/decorators.py

```diff
@@ -330,3 +335,33 @@ def pjm_update_dates(dates, args_dict):
             )
 
     return new_dates
+
+
+def ercot_update_dates(dates, args_dict):
+    date = args_dict["date"]
+    end = args_dict["end"]
+    years = {x for x in range(date.year, end.year + 1)}
+
+    fixed_dates = []
+
+    if date.year == end.year:
```

Comment:
i think in the case where this is true you can just return `dates`, since no modification is needed.

also, i would move this log up to the top of the function.